.storybook/test-data.js

```diff
@@ -324,7 +326,9 @@ const state = {
             },
           },
           options: {},
-          methods: [...Object.values(EthMethod)],
+          methods: [...Object.values(EthMethod)].filter(
```

Comment:
What about using the new `ETH_EOA_METHODS` constant here? (and actually everywhere we use this pattern)